src/app/Library/CrudPanel/CrudField.php

```diff
@@ -32,7 +42,7 @@ public function __construct($name)
             $this->setAttributeValue('name', $name);
         }
 
-        return $this->save();
+        $this->save();
```

Comment:
Did you intentionally remove the return here or was it a mistake? If so why? Thanks!

Reply:
`__construct` dont need return and cannot return, its void.

Reply:
Fair enough! 😄